### PR TITLE
Latest Release

### PR DESCRIPTION
**Pull request by TinasheMzondiwa**

### CODE REVIEW

File: common/translations/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -105,4 +105,6 @@
     <string name="ss_more">更多</string>
     <string name="ss_copyright">%1$s 基督复臨安息日會總會®</string>
     <string name="ss_see_all">看到所有</string>
+    <string name="ss_media_video">視頻</string>
+    <string name="ss_media_audio">" 聲音的"</string>
```
Comment (VitalikL) [inline common/translations/src/main/res/values-zh-rTW/strings.xml]: Seems like an extra space at the beginning

**Review by VitalikL — COMMENTED** [review]
One small issue with the translation (See below).

File: common/translations/src/main/res/values-zh-rTW/strings.xml (hunk 1 above)
Comment (TinasheMzondiwa) [reply common/translations/src/main/res/values-zh-rTW/strings.xml]: Fixed

**Review by VitalikL — APPROVED** [review]
🚀

**Pull request merged**